tensor2tensor/data_generators/wmt.py

```diff
@@ -267,6 +269,10 @@ def bi_vocabs_token_generator(source_path,
 # English-Czech datasets
 _ENCS_TRAIN_DATASETS = [
     [
```

Comment:
this doesn't seem to be a WMT file. maybe this makes sense as a new problem instead of adding onto wmt.

Reply:
The CzEng corpus is officially part of the WMT training data for English-Czech, see http://statmt.org/wmt17/translation-task.html.
Without CzEng you cannot get the SotA WMT scores (it is much bigger than news-commentary-v12.cs-en and europarl.cs-en).